packages/cli/src/commands/generate/function/__tests__/fixtures/jsFunc.js

```diff
@@ -1,18 +1,34 @@
+import { logger } from 'src/lib/logger'
```

Comment:
Am adding logger to enforce best practices since on of the security considerations will be visibility and one should log what the function is doing to see if an abuse.